index.js

```diff
@@ -224,12 +230,12 @@ class Plugin extends MiniAccountsPlugin {
         this._channelToAccount.set(existingChannel, account)
         await this._registerAutoClaim(account)
       } catch (e) {
-        debug('deleting channel because of failed validate. error=', e)
+        this._log.debug('deleting channel because of failed validate. error=', e)
         try {
           await this._channelClaim(account)
           account.deleteChannel()
         } catch (err) {
-          debug('could not delete channel. error=', err)
+          this._log.debug('could not delete channel. error=', err)
```

Comment:
There's something wrong if we can't delete a channel so I'd make this an `error`